src/Microsoft.SqlTools.ServiceLayer/SchemaDesigner/Contracts/GetSchemaModelRequest.cs

```diff
@@ -0,0 +1,24 @@
+//
+// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+//
+
+using Microsoft.SqlTools.Hosting.Protocol.Contracts;
+
+namespace Microsoft.SqlTools.ServiceLayer.SchemaDesigner
+{
+    public class GetSchemaModelRequestParams
+    {
+        public string OwnerUri { get; set; }
+        public string DatabaseName { get; set; }
```

Comment:
@cssuh is going to need an API as part of his Table Explorer work to get just the schema model for a view or table and the immediately adjacent schema objects.  Just a heads-up to look for him to reach out to you.